src/components/state.ts

```diff
@@ -6,7 +6,7 @@ export const LIST_WIDTH_COLLAPSED = 0;
 export const orientation = ref('');
 export const collapsed = ref(true);
 export const toggleList = () => (collapsed.value = false);
-export const closeList = () => ((collapsed.value = true), console.log(orientation));
+export const closeList = () => ((collapsed.value = true), (selectedMuscle.value = ''), console.log(orientation));
```

Comment:
nice use of the `,` operator, but consider using a normal function body instead when executing multiple statements: 
```js
() => {
    collapsed.value = true;
    selectedMuscle.value = '';
    ...
}

Reply:
done